andela/hc-millennium-falcon
# #156577923 Send Email alert when a job is run too often

**What does this PR do?**
This PR enables an Email alert to be sent to a user when a job is run too often and creates an icon on the dashboard for alerts that are run too often

**Description of Task to be completed?**
When a job is pinged too early, an Email alert is sent to the user and the status icon of the job changes to red.

**How should this be manually tested?**
1. Create a job with a timeout period of three minutes and grace period of one minute.
2. Ping the job URL to activate it
3. Ping that same URL again in less than a minute
4. An email alert should be sent to your mailbox
5. The icon on the dashboard next to the should change to red indicating it is running too often

**Any background context you want to provide?**
1. Previously a user was not able to be sent an E-mail when a job was pinged before the timeout period minus grace period. Now they can.
2. Previously jobs that were run too often would not be shown on the dashboard. Now they can

**What are the relevant pivotal tracker stories?**
[#156577923](https://www.pivotaltracker.com/story/show/156577923)
